### DISCUSSION TITLE
Query filter

### DISCUSSION THREAD

**Discussion by kodaizen** [WordPress/gutenberg#38047] [Q&A]
Hello,
Is it possible to limit the search result to posts belonging to the logged-in user only?
For the query block.

**Comment by t-hamano** [comment]
How about using `query_loop_block_query_vars` hook introduced in WordPress 6.1?

[Extending the Query Loop block | Block Editor Handbook](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/extending-the-query-loop-block/)
